apps/nextra/pages/en/build/smart-contracts/digital-asset.mdx

```diff
@@ -2,416 +2,321 @@
+</Callout>
 
-```move
-module 0x42::example {
-  use aptos_token_objects::collection;
-  use std::string::utf8;
+You can further customize your `Collection` by adding more resources or functionalities via smart contract. For example, a `Collection` can track when it was created in order to limit when `Token`s can be minted like so:
 
-  struct MyCollectionMetadata has key {
+```move filename="Example.move"
+struct MyCollectionMetadata has key {
     creation_timestamp_secs: u64,
-  }
+}
 
-  public entry fun create_collection(creator: &signer) {
+public entry fun create_collection(creator: &signer) {
     // Constructor ref is a non-storable struct returned when creating a new object.
     // It can generate an object signer to add resources to the collection object.
     let collection_constructor_ref = &collection::create_unlimited_collection(
-      creator,
-      utf8(b"My Collection Description"),
-      utf8(b"My Collection"),
-      royalty,
-      utf8(b"https://mycollection.com"),
+        creator,
+        "My Collection Description",
+        "My Collection",
+        royalty,
+        "https://mycollection.com",
     );
     // Constructor ref can be exchanged for signer to add resources to the collection object.
     let collection_signer = &object::generate_signer(collection_constructor_ref);
-    move_to(collection_signer, MyCollectionMetadata {
-      creation_timestamp_secs: timestamp::now_seconds()
-    })
-  }
+    move_to(collection_signer, MyCollectionMetadata { creation_timestamp_secs: timestamp::now_seconds() } })
 }
 ```
 
-### Token creation
+## Creating Tokens
+
+`Token`s have similar fields to `Collection`s: 
 
-Creators can mint tokens, which are separate objects from the collection. This
-allows for greater customization. Tokens can be created in two ways:
+- Description - an optional string smaller than 2048 characters (modifiable with a `MutatorRef`).
+- Name - a required string to identify the `Collection` that is unique within each `Collection`. This means a single `Collection` account cannot have more than one `Token` with the same name.
+- Royalty - an optional number indicating what % of the sale price goes to the creator of the `Collection`. This can be changed with a `MutatorRef` generated by the [Royalty module](https://github.com/aptos-labs/aptos-core/blob/main/aptos-move/framework/aptos-token-objects/sources/royalty.move) (an extension for the DA standard. See example usage in [`aptos_token.move`](https://github.com/aptos-labs/aptos-core/blob/main/aptos-move/framework/aptos-token-objects/sources/aptos_token.move)). Usually royalty is set on collections, but setting it on `Token`s allows the individual `Token` to have a custom royalty amount.
+- URI length - an optional string that is smaller than 512 characters which links to relevant content for the `Collection` (modifiable with a `MutatorRef`).
```

Comment:
Put this under `## Tokens`

```suggestion
| **Field** | **Description** |
|-----------|-----------------|
| **Description** | An optional string smaller than 2048 characters (modifiable with a `MutatorRef`). |
| **Name** | A required string to identify the `Collection` that is unique within each `Collection`. This means a single `Collection` account cannot have more than one `Token` with the same name. |
| **Royalty** | An optional number indicating what % of the sale price goes to the creator of the `Collection`. This can be changed with a `MutatorRef` generated by the [Royalty module](https://github.com/aptos-labs/aptos-core/blob/main/aptos-move/framework/aptos-token-objects/sources/royalty.move) (an extension for the DA standard. See example usage in [`aptos_token.move`](https://github.com/aptos-labs/aptos-core/blob/main/aptos-move/framework/aptos-token-objects/sources/aptos_token.move)). Usually royalty is set on collections, but setting it on `Token`s allows the individual `Token` to have a custom royalty amount. |
| **URI length** | An optional string that is smaller than 512 characters which links to relevant content for the `Collection` (modifiable with a `MutatorRef`). |

```